### DISCUSSION TITLE
German Umlauts "destroyed" when converting to PDF/A

### DISCUSSION THREAD

**Discussion by docbobo** [ocrmypdf/OCRmyPDF#1305] [Q&A]
I am trying to use OCRmyPDF to convert an existing PDF to PDF/A. Command looks like this

```
ocrmypdf --output-type pdfa --skip-text input.pdf output.pdf
```

In the resulting `output.pdf` German special characters are destroyed. When using `pdf` (not `pdfa`) as output type, the special characters are maintained correctly. Am I doing something wrong here?

**Comment by jbarlow83** [comment]
It should help to specify  `--language deu` (or any other languages that may appear) - otherwise the default is English and there is a bias in OCR to dropping characters/words unlikely to appear in the target language. 

Although, that shouldn't cause characters to be dropped on PDF/A.

One thing to try is use `--output-type pdf`, confirm special characters appear, then try rewriting the PDF using 
`gs -q -sDEVICE=pdfwrite -o out.pdf in.pdf`. This command should reconstruct an equivalent PDF. If that rewritten version drops characters, the issue may lie with Ghostscript, which handles PDF/A.

**Reply by tcurdt** [reply]
That's a litte confusing. I thought OCRmyPDF just adds a layer?
Why would it change existing characters, @jbarlow83 ?

**Comment by jbarlow83** [comment]
It's also possible one of the blacklisted Ghostscript versions was used (10.0.0 through 10.3.0 inclusive). These are known to corrupt output. 

Without a reproducible test case and information about versions used, I can only an educated guess.
